ansible/roles/cfg-logging/tasks/main.yml

```diff
@@ -7,7 +7,5 @@
 
 - name: Place logging config file
   template: src={{src}} dest={{dest}} owner={{owner}} group={{group}} mode=0644
-  when: "'{{group_name}}' in group_names"
```

Comment:
I think we need this protection in here (since the ansible/playbooks/10-services-cfg.yaml playbook uses the `services` host), so that configs for anon-files and kifshare don't get deployed in our other services host, and all our other service configs don't deployed on our anon-files/kifshare host, etc.